ui/dashboards/src/context/TimeRangeProvider/TimeRangeProvider.tsx

```diff
@@ -28,15 +28,27 @@ export interface TimeRangeProviderProps {
 export function TimeRangeProvider(props: TimeRangeProviderProps) {
   const { initialTimeRange, enabledURLParams, children } = props;
```

Comment:
Curious can you check if setting enabledURLParams to false still works?

Reply:
yep still works!